go.mod

```diff
@@ -126,6 +126,8 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
+replace github.com/Masterminds/squirrel v1.5.2 => github.com/lieut-data/squirrel v1.5.4
```

Comment:
I've forked this into my own organization for now, but open to promoting it to `mattermost/squirrel` if we think that makes sense!

Reply:
I think we should do that. There are more improvements in squirrel to make. :) But need not block this PR.